docs/docs/configuring-ca-certificates.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Configuring CA Certificates
+---
+
+For developers using a private (or corporate) package registry that requires a certificate from a certificate authority (CA), you may need to set your certificate in your npm, yarn, or node config.
+
+## Common errors from misconfigured certificates
+
+This might be a problem if you are seeing errors like `unable to get local issuer certificate` in the console output while trying to install Gatsby plugins, particularly with plugins or themes that need to be built as native Node.js modules (eg. `gatsby-plugin-sharp`). It may happen after running `npm install` or `yarn install` in a fresh clone of a repository when trying to pull packages from a place besides a public registry and the certifcate has not been set in your config.
```

Comment:
```suggestion
If you're seeing errors like `unable to get local issuer certificate` in the console output while trying to install a Gatsby plugin, a misconfigured certificate might be the problem. This occurs particularly with plugins or themes that need to be built as native Node.js modules (eg. `gatsby-plugin-sharp`). It may happen when installing packages from a private registry (via `npm install` or `yarn install`) without an appropriately setup certificate in config.
```